reactive/kotlinx-coroutines-reactor/test/FluxTest.kt

```diff
@@ -23,7 +22,7 @@ class FluxTest : TestBase() {
         expect(2)
         flux.subscribe { value ->
             expect(5)
-            Assert.assertThat(value, IsEqual("OK"))
+            Assert.assertEquals("OK", value)
```

Comment:
`Assert.` prefix is wrong and should be removed.

Reply:
Do you mean that `assertEquals` should be taken from `kotlin.test` or that `Assert` should be part of the import at the top of the file?